editor/editor_node.cpp

```diff
@@ -584,18 +586,6 @@ void EditorNode::_notification(int p_what) {
 
 			ResourceImporterTexture::get_singleton()->update_imports();
 
-			if (settings_changed) {
-				_update_title();
-			}
-
-			if (settings_changed) {
-				_update_from_settings();
-				settings_changed = false;
-				emit_signal(SNAME("project_settings_changed"));
-			}
-
-			ResourceImporterTexture::get_singleton()->update_imports();
```

Comment:
I'm not sure why was this called second time here and where to put it after the changes.